backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/database/jpa/PostgresDocumentTypeRepositoryImpl.java

```diff
@@ -30,6 +30,25 @@ public List<DocumentType> findCaselawBySearchStr(String searchString) {
         .toList();
   }
 
+  @Override
+  public List<DocumentType> findIndependentLiteratureBySearchStr(String searchString) {
+    return repository
+        .findCaselawBySearchStrAndCategory(
+            searchString, categoryRepository.findFirstByLabel("U").getId())
+        .stream()
+        .map(DocumentTypeTransformer::transformToDomain)
+        .toList();
+  }
+
+  @Override
+  public List<DocumentType> findAllIndependentLiteratureOrderByAbbreviationAscLabelAsc() {
+    return repository
+        .findAllByCategoryOrderByAbbreviationAscLabelAsc(categoryRepository.findFirstByLabel("U"))
+        .stream()
+        .map(DocumentTypeTransformer::transformToDomain)
+        .toList();
```

Comment:
@leonie-koch Just one note on the method names: I believe U=Unselbstständige Literatur would be DependentLiterature (~Abhängige Literatur) instead of Independent, right?

Reply:
absolutely right, this translation really messed with my mind (INdependent != UNabhängig) :D i fixed that